website/client-old/js/services/taskServices.js

```diff
@@ -373,6 +373,8 @@ angular.module('habitrpg')
 
         summary += ' on the ' + (week + 1) + ' ' + longDay;
       }
+      
+      if (task._edit.everyX === 0) summary = 'Repeats never.';
```

Comment:
Can you change this to a translatable string. Also, you can but this condition at the top and simple return the repeats never string.